### PR TITLE
Fixed inability to disable combo loop check

### PR DESCRIPTION
**Pull request by chick**
Moved checking of dontCheckComboLoops into loadAnnotations so
that it works in cases where Driver.execute is not used.
This is part of a fix for [--no-check-comb-loops doesn't work](https://github.com/freechipsproject/chisel-testers/issues/155)

### CODE REVIEW

**Comment by jackkoenig** [comment]
Can you fix the failing test?

**Comment by chick** [comment]
retest this please

**Comment by chick** [comment]
retest this please

**Comment by chick** [comment]
@jackkoenig tests all pass for me locally, when I look at the jenkins log it looks like error is in chisel. So this is probably failing because it has to be paired with a change to chisel testers?

File: src/test/scala/firrtlTests/DriverSpec.scala [hunk 1]
```diff
@@ -137,7 +137,8 @@ class DriverSpec extends FreeSpec with Matchers with BackendCompilationUtilities
     copyResourceToFile("/annotations/SampleAnnotations.anno", annotationsTestFile)
     optionsManager.firrtlOptions.annotations.length should be (0)
     Driver.loadAnnotations(optionsManager)
-    optionsManager.firrtlOptions.annotations.length should be (10) // 9 from circuit plus 1 for targetDir
+    println(s"annotations ${optionsManager.firrtlOptions.annotations.mkString("\n")}")
```
Comment (ucbjrl) [inline src/test/scala/firrtlTests/DriverSpec.scala]: Looks like a debug println.

**Pull request merged**